com.umlet.Umlet.yml

```diff
@@ -0,0 +1,48 @@
+app-id: com.umlet.Umlet
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+
+rename-icon: umlet_logo
+rename-desktop-file: umlet.desktop
+rename-appdata-file: metainfo.xml
+command: umlet
+
+finish-args:
+    - --share=ipc
+    - --socket=x11
+    - --device=dri
+    - --filesystem=home
+    - --filesystem=xdg-config/fontconfig:ro
+    - --env=PATH=/app/jre/bin:/usr/bin:/app/bin
+    - --env=UMLET_HOME=/app/lib/umlet
+
+modules:
+    - name: openjdk
+      buildsystem: simple
+      build-commands:
+        - /usr/lib/sdk/openjdk11/install.sh
+
+    - name: Umlet
+      buildsystem: simple
+      build-commands:
+        - install -D -m755 umlet.jar -t /app/lib/umlet
+        - cp -a {lib,palettes,custom_elements} /app/lib/umlet
+        - install -D -m755 umlet.sh /app/bin/umlet
+        # fix start command
+        - sed -i -e 's/Exec=java -Dsun.java2d.xrender=f -jar umlet.jar/Exec=umlet/g' umlet.desktop
+        - install -D -m644 umlet.desktop -t /app/share/applications
+        - install -D -m644 umlet_logo.png -t /app/share/icons/hicolor/512x512/apps
+      post-install:
+        - install -D -m644 metainfo.xml -t /app/share/metainfo
+      sources:
+        - type: archive
+          url: https://www.umlet.com/download/umlet_15_0/umlet-standalone-15.0.0.zip
+          sha256: 81dbe1a981b2ac5b90861ae4176eb05dbdd340b4422e6e7dfee4b14cf9877401
+        - type: file
+          url: https://raw.githubusercontent.com/umlet/umlet/0da4d7d175fd52283ed3a5e8fe38d4723e7ac51f/umlet-standalone/src/exe/img/umlet_logo.png
```

Comment:
this image is in the zip. there is no point downloading it separately.

Reply:
fixed

Reply:
ok I had to revert this, because the icon in the zip is not squared, because the commit which fixes this problem happened after the release of the version.

So I have to download it separately until upstream published a new version which contains that fix.

Reply:
ah ok.